CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Mapbox welcomes participation and contributions from everyone.
 * The behavior of `GestureManager.options` has been updated to better reflect the `isEnabled` state of the associated gesture recognizers. ([#677](https://github.com/mapbox/mapbox-maps-ios/pull/677))
 * The gesture recognizer properties of `GestureManager` are no longer `Optional`. ([#677](https://github.com/mapbox/mapbox-maps-ios/pull/677))
 * `GestureType` has been redesigned so that its cases have a 1-1 relationship with the built-in gestures. ([#677](https://github.com/mapbox/mapbox-maps-ios/pull/677))
+* `GestureManagerDelegate.gestureBegan(for:)` has been renamed to `GestureManagerDelegate.gestureManager(_:didBegin:)`. Added the public delegate methods ``GestureManagerDelegate.gestureManager(_:didEnd:willDecelerate:)` and `GestureManagerDelegate.gestureManager(_:didEndDeceleratingFor:)`. ([#697](https://github.com/mapbox/mapbox-maps-ios/pull/697))
```

Comment:
Maybe split this changelog into two lines. The "Added the public delegate methods..." should be a separate entry.

Also looks like there is an issue with back ticks here